Updates moose-test to use new constructors

aeslaughter opened the pull request:
(refs #4606)

aeslaughter committed:
Updates moose-test to use new constructors

(refs #4606)

moosebuild commented:
Results of testing 6cf3b0a1ceee828be40d0d35499446b081e33721 using moose_PR_pre_check recipe:

Passed on: linux-gnu

View the results here: https://www.moosebuild.org/view_job/19425

moosebuild commented:
Results of testing 6cf3b0a1ceee828be40d0d35499446b081e33721 using moose_PR_app_tests recipe:

Failed on: linux-gnu

View the results here: https://www.moosebuild.org/view_job/19428

moosebuild commented:
Results of testing 6cf3b0a1ceee828be40d0d35499446b081e33721 using moose_PR_test recipe:

Passed on: linux-gnu

View the results here: https://www.moosebuild.org/view_job/19426

moosebuild commented:
Results of testing 6cf3b0a1ceee828be40d0d35499446b081e33721 using moose_PR_test_dbg recipe:

Passed on: linux-gnu

View the results here: https://www.moosebuild.org/view_job/19427

moosebuild commented:
Results of testing 6cf3b0a1ceee828be40d0d35499446b081e33721 using moose_PR_app_tests recipe:

Failed on: linux-gnu

View the results here: https://www.moosebuild.org/view_job/19428

moosebuild commented:
Results of testing 63461319035218f38ebb16df5a43d5a051a61da1 using moose_PR_pre_check recipe:

Passed on: linux-gnu

View the results here: https://www.moosebuild.org/view_job/19467

moosebuild commented:
Results of testing 63461319035218f38ebb16df5a43d5a051a61da1 using moose_PR_app_tests recipe:

Failed on: linux-gnu

View the results here: https://www.moosebuild.org/view_job/19470

moosebuild commented:
Results of testing 63461319035218f38ebb16df5a43d5a051a61da1 using moose_PR_test recipe:

Passed on: linux-gnu

View the results here: https://www.moosebuild.org/view_job/19468

moosebuild commented:
Results of testing 63461319035218f38ebb16df5a43d5a051a61da1 using moose_PR_test_dbg recipe:

Passed on: linux-gnu

View the results here: https://www.moosebuild.org/view_job/19469

aeslaughter committed:
Make _real_zero const in ZeroInterface

In order for the InputParameters to be a const reference, the PostprocessorInterface needs to return const

references, which as reported in #2381 is problematic. Changing the ZeroInterface to hold a const reference seems like
it may fix the problem, which is manifested in BISON.
(refs #4606, #2381)

moosebuild commented:
Results of testing 5fdee891dd59ff712b695695287683bc169fddf1 using moose_PR_pre_check recipe:

Passed on: linux-gnu

View the results here: https://www.moosebuild.org/view_job/19503

moosebuild commented:
Results of testing 5fdee891dd59ff712b695695287683bc169fddf1 using moose_PR_app_tests recipe:

Failed on: linux-gnu

View the results here: https://www.moosebuild.org/view_job/19506

moosebuild commented:
Results of testing 5fdee891dd59ff712b695695287683bc169fddf1 using moose_PR_test recipe:

Passed on: linux-gnu

View the results here: https://www.moosebuild.org/view_job/19504

moosebuild commented:
Results of testing 5fdee891dd59ff712b695695287683bc169fddf1 using moose_PR_test_dbg recipe:

Passed on: linux-gnu

View the results here: https://www.moosebuild.org/view_job/19505

aeslaughter commented:
This requires an integration branch, I will open a new one.
